cryptographic_estimators/helper.py

```diff
@@ -60,20 +63,20 @@ def concat_pretty_tables(t1: str, t2: str):
 
 def _truncate(x: float, precision: int):
     """
-    truncate a value
+    Truncate a value
 
     INPUT:
 
     - ``x`` -- value to truncate
     - ``precision`` -- number of decimial digits to truncate to
 
     """
-    return float(int(x * 10 ** precision) / 10 ** precision)
+    return float(int(x * 10**precision) / 10**precision)
 
 
 def round_or_truncate(x: float, truncate: bool, precision: int):
     """
-    eiter rounds or truncates `x` if `truncate` is `true`
+    Eiter rounds or truncates `x` if `truncate` is `true`
```

Comment:
Either

Reply:
Huh, I missed that error.  Thanks! I will fix this typo in my next PR, so we can merge this right now.